### PR TITLE
Add prices tags wrapped with spans

### PR DESCRIPTION
**Pull request by artursmet**

### CODE REVIEW

File: django_prices/templatetags/prices_wrapped.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    locale_code = to_locale(language)
+    if not localedata.exists(locale_code):
+        localedata.load(locale_code)
+    locale = Locale(locale_code)
+    currency_format = locale.currency_formats.get(None)
+    pattern = currency_format.pattern
+    pattern = re.sub(
+        '(\xa4+)', '<span class="currency">\\1</span>', pattern)
+    pattern = re.sub(
+        '(#,##0.00)', '<span class="amount">\\1</span>', pattern)
```
Comment (patrys) [inline django_prices/templatetags/prices_wrapped.py]: This is not likely to match all locales, is it?

Comment (artursmet) [inline django_prices/templatetags/prices_wrapped.py]: I tested it for various combinatons of locale and currency and it looks like it's working as expected.

Comment (patrys) [inline django_prices/templatetags/prices_wrapped.py]: I think at least India and China do not use s similar pattern. Please confirm.

**Comment by artursmet** [comment]
As we discussed - I changed behavior of template tag, to wrap only currency in spans. Please review @mociepka

**Comment by patrys** [comment]
Please implement this as a `format` parameter that can be passed to the existing template tags (if we want the default to be `html` then we will need to bump the version major).

**Comment by artursmet** [comment]
@patrys I implemented it now as `html` parameter

**Pull request merged**